sites/friday/src/get-started/express/add-database.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Add a database"
+weight: -130
+description: |
+  Once Your Express application has been deployed on {{% vendor/name %}}, you might want to add a service to your application.
+---
+
+{{% description %}}
+
+{{% vendor/name %}} projects already include a [variety of services](/add-services.html#available-services) so you don’t have to subscribe to an external cache or search-engine services.
+And as these services are included in your project, you can manage them through Git and they’re backed up along with the rest of your project.
+Your project source code defines the services configuration in the main `.{{% vendor/cli %}}/config.yaml` file and this is where you can add new services.
+
+As an example on how to do so, to add a [MariaDB database engine](/add-services/mysql.html) into your Express project, complete the following 4 steps:
+
+Configure the MariaDB service by adding this Yaml definition at the end of your `.{{% vendor/cli %}}/config.yaml` file:
+
+```yaml
+# .{{% vendor/cli %}}/config.yaml
+services:
+  db:
+    type: mariadb:10.4
+```
+
+Connect the service to your application by adding a relationships settings into your app definition, within your .{{% vendor/cli %}}/config.yaml file:
+```yaml
+# .{{% vendor/cli %}}/config.yaml
+applications:
+  app:
+    type: nodejs:18
+    # The relationships of the application with services or other applications.
+    # The left-hand side is the name of the relationship as it will be exposed
+    # to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+    # side is in the form `<service name>:<endpoint name>`.
+    # More information: {{% vendor/url_doc %}}create-apps/app-reference.html#relationships
+    relationships:
+      database:
+        service: "db"
+        endpoint: "mysql"
+```
+
+Configure your Express project to use it following the [official documentation](https://expressjs.com/en/guide/database-integration.html#mysql).
```

Comment:
we should expand this to include how they can include this information via environmental variables instead of hard-coding the information from the output of `upsun relationships` 
Possibly reusing this piece from the current express template?

Reply:
i need to finish this part `Add a database` of the guide. 
At this time, only the first step of the guide is valid (no more TODO in it)
I will do it this afternoon!